superset-frontend/src/components/TableCollection/index.tsx

```diff
@@ -31,6 +31,7 @@ interface TableCollectionProps {
   columns: TableInstance['column'][];
   loading: boolean;
   highlightRowId?: number;
+  columnsForWrapText?: any[];
```

Comment:
```suggestion
  columnsForWrapText?: string[];
```

Reply:
I tired it with **string** type but it gave the type issue to me even though type of cloum.Header value is **string**.
And so I had used to use **any** type.

Reply:
@evans 

Resolved.

Reply:
@evans

Resolved.